core/templates/core/Order_detail.html

```diff
@@ -0,0 +1,123 @@
+              NO items found
+              {% endif %}
+            </h5>
+          </div>
+          <!-- ------------------------------------------------------- -->
+          {% for product in object_list %}
+          <div class="card mb-4">
+            <div class="card-body">
+
+              <!-- Single item -->
+              <div class="row">
+                <div class="col-lg-3 col-md-12 mb-4 mb-lg-0">
+                  <!-- Image -->
+                  <div class="bg-image hover-overlay hover-zoom ripple rounded" data-mdb-ripple-color="light">
+                    <img src="{{ product.product.image.url }}" class="w-100" />
+                    <a href="#!">
+                      <div class="mask" style="background-color: rgba(251, 251, 251, 0.2)"></div>
+                    </a>
+                  </div>
+                  <!-- Image -->
+                </div>
+
+                <div class="col-lg-5 col-md-6 mb-4 mb-lg-0">
+                  <!-- Data -->
+                  <div>
+                    <p class="font-weight-bold" style="margin:0;display:inline;">Name:</p>
+                    <p style="display:inline;">{{ product.product.name }}</p>
+                  </div>
+                  <div>
+                    <p class="font-weight-bold" style="display:inline;">Quantity:</p>
+                    <p style="display:inline;" id="stock">{{ product.quantity }}</p>
+                  </div>
+                  <div>
+                    <p class="font-weight-bold" style="display:inline;">Discription:</p>
+                    <p style="display:inline;">{{ product.product.description }}</p>
+                  </div>
+                  <div>
+                    <p class="font-weight-bold" style="display:inline;">Price:</p>
+                    <p style="display:inline;">
+                      <span> <strong>$</strong><strong name="price">{{ product.price }}</strong></span>
+                    </p>
+                  </div>
+                  <!-- Data -->
+                </div>
+
+                <div class="col-lg-4 col-md-6 mb-4 mb-lg-0 text-center font-weight">
+                  <!-- Quantity -->
+                  
+                </div>
+              </div>
+            </div>
+          </div>
+          {% endfor %}
+          <!--  ------------------------------------------------------------------------------ -->
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+{% endblock %}
+
+<!-- Footer -->
+{% block footer %}
+<footer>
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <div class="inner-content">
+          <p>Copyright &copy; 2020.
+        
+        - Design: <a rel="nofollow noopener" href="#" target="_blank">Betatesting</a></p>
+        </div>
+      </div>
+    </div>
+  </div>
+</footer>
+{% endblock %}
+
+{% block javascript_tag %}
+
+<script>
+  // Get total price of products
+
+
+  $(document).ready(function () {
+    if (isLogin) {
```

Comment:
is this code intended or was it for testing?

Reply:
It is for testing. And I remove this code.